apps/condo/index.js

```diff
@@ -18,7 +18,7 @@ const bodyParser = require('body-parser')
 const { formatError } = require('@condo/domains/common/utils/apolloErrorFormatter')
 
 const IS_ENABLE_DD_TRACE = conf.NODE_ENV === 'production'
-const IS_ENABLE_APOLLO_DEBUG = conf.NODE_ENV === 'development' || conf.NODE_ENV === 'test'
+const IS_ENABLE_APOLLO_DEBUG = conf.DEBUG_APOLLO && conf.NODE_ENV !== 'production'
```

Comment:
For what use-case Apollo debugging mode is enabled only for non-production environment?
How that extra `uid` field will help me to debug an application in `development` and `test` node environments?

Reply:
@AntonAL 
Whad do you mean about uid?